project/Dependencies.scala

```diff
@@ -177,28 +186,22 @@ object Dependencies {
       akka.persistenceQuery       % Compile,
       akka.projectionEventSourced % Compile,
       mongodb.scalaDriver         % Compile,
-      spray.spray                 % Compile
+      spray.spray                 % Compile,
+      cats.core                   % Compile
     )
 
     lazy val rateLimiterDependencies: Seq[ModuleID] =
       Seq(
         redis.jedis        % Compile,
+        akka.stream        % Compile,
         spray.spray        % Compile,
+        cats.core          % Compile,
         scalamock.core     % Test,
         akka.httpTestkit   % Test,
         akka.streamTestkit % Test
       )
 
-    lazy val commonDependencies: Seq[ModuleID] = Seq(
-      // For making Java 12 happy
-      "javax.annotation"         % "javax.annotation-api" % "1.3.2" % "compile",
-      apacheCommons.text         % Compile,
-      atlassian.swaggerValidator % Compile,
-      cats.core                  % Compile,
-      logback.classic            % Compile,
-      typesafe.config            % Compile,
-      lightbend.logging          % Compile,
-      scalatest.core             % Test
-    )
+    lazy val commonDependencies: Seq[ModuleID] =
+      Seq(aws.sts % Runtime, logback.classic % Runtime, scalatest.core % Test)
```

Comment:
I would not add sts here because it means that aws dependency is included in the service even if it does not use aws sdk.
May we leave this only where it is required? 🙏

Reply:
I thought that I was required everywhere to be fair, but I'll add it where needed

Reply:
no no, this is required only when the service uses the aws sdk.
If you are referring to IAM roles for kubes, they are handled directly in the deployment 👍

Reply:
👍 I see that just file and signer should be using it though

Reply:
⚠️ also the queue manager 🙏 
basically, everywhere the aws sdk is used